### PR TITLE
just some cleanup

### PR DESCRIPTION
**Pull request by terminaldweller**
literally just that, removed step 1111 from the test function since it doesnt exist in the expectations.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -50,13 +50,13 @@ then use `geth attach ipc:///tmp/ethereum_dev_mode/geth.ipc` with
 var account = personal.newAccount('')
 account
 // and copy the account to below
-personal.unlockAccount(account)
+personal.unlockAccount(account, '', 1000000)
 miner.setEtherbase(account)
-miner.start()
+miner.start(n)
```
Comment (chriseth) [inline test/index.js]: Is `n` defined somewhere?

Comment (terminaldweller) [reply test/index.js]: no, by n i just meant pick a number.

Comment (chriseth) [reply test/index.js]: I would like to copy-paste the code, and in my geth, no argument is required.

Comment (terminaldweller) [reply test/index.js]: ok, ill just remove it

**Pull request merged**